src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -432,8 +439,8 @@ def get_max_segs_crs(pool_results):
 
 
 def compute_slices_for_starmap(
-    ramp_data, buffsize, save_opt, readnoise_2d, gain_2d, weighting, number_slices
-):
+        ramp_data, buffsize, save_opt,
+        readnoise_2d, gain_2d, weighting, avg_dark_current, number_slices):
```

Comment:
Need to add `avg_dark_current` to list of parameter definitions below. Or does it no longer need to be part of this function call, since it's been moved to be part of the ramp data class?

Reply:
It's been removed.